src/screens/OtherProfile/index.js

```diff
@@ -628,14 +550,14 @@ const OtherProfile = () => {
           data={handleDataFeed()}
           onScroll={handleScroll}
           onRefresh={handleRefresh}
-          refreshing={loading}
+          refreshing={isLoading}
           onMomentumScrollEnd={(event) => {
             onWillSendViewPostTime(event, feeds);
             fetchNextFeeds(event);
           }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The state `loading` has been replaced with `isLoading`. Ensure that `isLoading` is equivalent to `loading` to avoid any behavioral changes.



<!-- This is an auto-generated comment by OSS CodeRabbit -->